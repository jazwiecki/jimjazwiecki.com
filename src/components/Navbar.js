import React from 'react'
import { Link } from 'gatsby'
import github from '../img/github-icon.svg'
import logo from '../img/logo.svg'

const Navbar = () => (
  <nav className="navbar is-transparent">
    <div className="container">
      <div className="navbar-brand">
        <Link to="/" className="navbar-item">
          <figure className="image">
            <img src={logo} alt="jazwiecki" class="logo" />
          </figure>
        </Link>
      </div>
      <div className="navbar-end">
        <a
          className="navbar-item"
          href="https://github.com/jazwiecki"
          target="_blank"
          rel="noopener noreferrer"
        >
          <span className="icon">
            <img src={github} alt="Github" />
          </span>
        </a>
      </div>
    </div>
  </nav>
)

export default Navbar
