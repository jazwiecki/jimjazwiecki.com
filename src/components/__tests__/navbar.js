import React from "react"
import renderer from "react-test-renderer"

import Navbar from "../Navbar"

// @todo write another component test

describe("Navbar", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Navbar />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
});