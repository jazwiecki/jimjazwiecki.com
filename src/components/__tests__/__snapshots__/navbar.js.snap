// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<nav
  className="navbar is-transparent"
>
  <div
    className="container"
  >
    <div
      className="navbar-brand"
    >
      <a
        className="navbar-item"
        href="/"
      >
        <figure
          className="image"
        >
          <img
            alt="jazwiecki"
            class="logo"
            src="test-file-stub"
          />
        </figure>
      </a>
    </div>
    <div
      className="navbar-end"
    >
      <a
        className="navbar-item"
        href="https://github.com/jazwiecki"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="icon"
        >
          <img
            alt="Github"
            src="test-file-stub"
          />
        </span>
      </a>
    </div>
  </div>
</nav>
`;
